CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.6.0
+
+> ## Note: Changes to the Zed lake storage format
+> 
+> This release includes a change to the Zed lake storage format that is **not**
+> backward compatible. To migrate your pools to the new format there is a 
+> [Zed lake migration](https://github.com/brimdata/zed-lake-migration) kit and
+> [specific guidance for users of the Zed CLI tools](https://github.com/brimdata/zed-lake-migration#zed-cli-tools).
+
+* Zed lake storage format is now at v2 (#4386, #4415)
```

Comment:
```suggestion
* Zed lake storage format is now at version 3 (#4386, #4415)
```

Reply:
Ah thanks. I had not noticed that the reported version string did indeed get advanced again.

```
$ zed -version
Version: v1.5.0-60-g739e2713

$ zed -lake lake serve
unsupported lake version: found version 1 while expecting 3
```